virttest/libvirt_xml/vm_xml.py

```diff
@@ -2127,13 +2127,13 @@ class CellDistancesXML(base.LibvirtXMLBase):
         Distances of cell
         """
 
-        __slots__ = ('sibling')
+        __slots__ = ('sibling',)
```

Comment:
I assume line 2130 and line 2136 changes are not necessary.

Reply:
It's necessary for 2130 since the only item in tumple can't be recognized in current version. For 2136, it's to make it unified with the definition in 2130 although it can work before.  Thanks